### PR TITLE
Fix alipay example of ShouKuanMa (收款码) and normalize outputs

### PR DESCRIPTION
**Pull request by oxalica**
1. Using the example config, income from ShouKuanMa are translated to expenses, which is incorrect. This PR add another rule in config to fix it.
2. The order of `open` directives change every time, since go's map is unordered. This PR sort it to make it deterministic.
3. Normal transactions output an empty line with only a tab. This PR remove it.

### CODE REVIEW

File: pkg/compiler/beancount/compiler.go [hunk 1]
```diff
@@ -111,10 +111,15 @@ func (b *BeanCount) writeHeader(file *os.File) error {
...
-		}
+    var sortedAccounts []string
+    for k := range accounts {
+        if k != "" {
+            sortedAccounts = append(sortedAccounts, k)
+        }
+    }
+    sort.Strings(sortedAccounts)
+
+	for _, k := range sortedAccounts {
```
Comment (Triple-Z) [inline pkg/compiler/beancount/compiler.go:122]: The indent for line 114-120 and 122 are different. Can u change the indent to which file original used(I think is tab)?

**Comment by Triple-Z** [comment]
@oxalica Thanks for your contribution!

**Comment by oxalica** [comment]
Fixed. Also added an example of normal transfer income.

**Review by Triple-Z — APPROVED** [review]
LGTM

**Pull request merged**